<launch> 
        <node name="sound_play_node" pkg="sound_play" type="soundplay_node.py"/>
        <node name="voice_broadcaster_node" pkg="voice_broadcaster" type="voice_broadcast_node.py"/>
</launch>